dxw/govuk-components-plugin
# Upgrade `iguana` for PHP 8 compatibility

This PR resolves the deprecation warning, by upgrading `iguana` to the latest version:
```
Return type of Dxw\Iguana\Value\ArrayBase::offsetUnset($offset) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice
```

## How to Test
1. Install and activate this plugin
2. Navigate to `wp-content/plugins/govuk-components-plugin`
3. Checkout this branch `chore/update-composer-dependencies`
4. Confirm the deprecation warning is no longer present
